host/platform/Windows/capture/DXGI/src/dxgi.c

```diff
@@ -776,9 +777,10 @@ static void computeFrameDamage(Texture * tex)
     };
 
     rectToFrameDamageRect(&moveRect->DestinationRect, texDamageRect++);
+    moved += 2;
   }
 
-  tex->damageRectsCount = dirtyRectsCount + moveRectsCount;
+  tex->damageRectsCount = dirtyRectsCount + moved;
```

Comment:
Shouldn't this be `2 * moveRectsCount` instead? This loop should never break early, because the size of the buffer was already checked.

Reply:
No, because you skip rectangles that haven't actually moved.

Reply:
Renamed variable to be more clear.